crates/rspack_plugin_rsdoctor/src/module_graph.rs

```diff
@@ -152,7 +152,7 @@ pub fn collect_module_original_sources(
         .or_else(|| {
           let resource = Utf8PathBuf::from(resource);
           let buffer = ifs.read_sync(&resource).ok()?;
-          let content = String::from_utf8(buffer).ok()?;
+          let content = String::from_utf8(buffer.to_vec()).ok()?;
```

Comment:
should use [.try_into_mut()](https://docs.rs/bytes/latest/bytes/struct.Bytes.html#method.try_into_mut) + [.into()](https://docs.rs/bytes/latest/bytes/struct.BytesMut.html#impl-From%3CBytesMut%3E-for-Vec%3Cu8%3E)